src/python_testing/TC_CNET_4_3.py

```diff
@@ -115,7 +115,7 @@ async def test_TC_CNET_4_3(self):
         self.step(4)
         if not current_cluster_connected:
             logging.info("Current cluster is not connected, skipping all remaining test steps")
-            self.skip_all_remaining_steps()
+            self.mark_all_remaining_steps_skipped()
```

Comment:
```suggestion
            self.mark_all_remaining_steps_skipped(5)
```
I just realized that I call these methods in a correct way. Is this ok we fix it here?